csharp/src/Apache.Arrow/Arrays/BinaryArray.cs

```diff
@@ -171,6 +171,7 @@ public TBuilder Clear()
         public ReadOnlySpan<byte> Values => ValueBuffer.Span.CastTo<byte>();
 
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        [Obsolete("This method has been deprecated. Please use ValueOffsets instead.")]
```

Comment:
How about `ValueOffsets[index]` instead of `ValueOffsets`?

Reply:
Sounds good.
I fixed it.